src/animation/interpolants/LinearInterpolant.js

```diff
@@ -0,0 +1,44 @@
+/**
+ *
+ * Interpolant the returns a time-proportional mix of the keyframe values of
+ * the surrounding interval.
+ *
+ *
+ * @author tschw
+ */
+
+THREE.LinearInterpolant = function( times, values, stride, result ) {
+
+	THREE.Interpolant.call( this, times, values, stride, result );
+
+};
+
+Object.assign( THREE.LinearInterpolant.prototype, THREE.Interpolant.prototype, {
+
+	constructor: THREE.LinearInterpolant,
+
+	_interpolate: function( i1, t0, t, t1 ) {
```

Comment:
I'd prefer descriptive variable names.  As the original author of the mixer, if I have to think about what these are for a bit, then it is likely non-obvious.  I am assuming it is index1, time0, time, time1?  I guess that time0 and time1 are the times of the surrounding keys and time is the current time?  I'd suggest somethingl along the lines of: keyTime0, keyTime1, keyIndex1, currentTime.